springfox-bean-validators/src/test/java/springfox/bean/validators/plugins/models/PatternTestModel.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *
+ *  Copyright 2015 the original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *
+ */
+package springfox.bean.validators.plugins.models;
+
+import javax.validation.constraints.Pattern;
+
+/**
+ * @author : ashutosh 
+ * 29/04/2016
+ */
+public class PatternTestModel {
```

Comment:
Probably it would be good to add here an example when we have both annotations `@Size` and `@Pattern` assigned to one field and check that both information is kept in the context? This might help to understand to somebody who will be interesting how it should work in this specific case. How do you think?